server/src/api/roadmaps/roadmaps.routes.ts

```diff
@@ -18,46 +18,51 @@ import customerRouter from '../customer/customer.routes';
 import rolesRouter from '../roles/roles.routes';
 import invitationsRouter from '../invitations/invitations.routes';
 
-const roadmapRouter = new KoaRouter<IKoaState, Context>();
+const roadmapRouter = new KoaRouter<IKoaState, Context>({
+  prefix: '/roadmaps',
+});
 
 roadmapRouter.use(requireAuth);
-roadmapRouter.use('/roadmaps/:roadmapId', requireRole);
+roadmapRouter.use('/:roadmapId', requireRole);
 
-roadmapRouter.get('/roadmaps', getRoadmaps);
-roadmapRouter.post('/roadmaps', postRoadmaps);
+roadmapRouter.get('/', getRoadmaps);
```

Comment:
Why get roadmaps -route does not require verified email? I do not see why an unverified account would need access to this route, since they are not in any roadmap

Reply:
Existing account's email may also become unverified by changing it, or possibly it could be set as unverified if it starts to generate bounces.

Also it otherwise gets stuck on a loading spinner on home page, before getting to the user info page.

Reply:
Thanks for explaining, looks like this is a well thought decision